gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "3.7.0",
+  "version": "3.8.0",
```

Comment:
This is just a documentation update so should be a patch version

Reply:
> This is just a documentation update so should be a patch version

De-bumped: https://github.com/sonatype/sonatype-react-shared-components/pull/313/commits/d03f60260740793e3b6c4fd01e41d55ead98257f